docs/organizer-guide/meetups/starting.rst

```diff
@@ -1,11 +1,10 @@
 Starting a Meetup
 =================
 
-Interested in fostering a lovely gathering of documentarians in your neck of the woods?
-This is the place to start!
+If you've been to a Write The Docs conference, you may have the need to "continue the conversation".
+With a little help, you can make it happen with a Meetup in your area!
 
-The first thing you'll want to do is watch this video about starting a meetup.
-It contains a lot of the knowledge that we have put here:
+Start by watching this video:
 
```

Comment:
I didn't change anything here from the original, but I don't see a video link in the "View". 

I'm not sure what I'm missing.